src/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -0,0 +1,13 @@
+type BackgroundImageProps = {
+  children: React.ReactNode;
+};
+
+const BackgroundImage = ({ children }: BackgroundImageProps) => {
+  return (
+    <div className="h-[960px] w-full bg-[url('/images/AK_bg_photo1.png')] bg-cover bg-center bg-no-repeat">
```

Comment:
Add a classname prop to allow users to extend or override the default styles

Reply:
There is no need for className prop in my opinion because the style classes will be in the children area of component. The styling in this component will be same for any other image.